common/src/main/java/org/wso2/testgrid/common/config/ConfigurationContext.java

```diff
@@ -267,7 +267,12 @@ public static boolean isDeploymentTinkererEnabled() {
         /**
          * Repeateble json string section of all logs
          */
-        REPEATABLE_ALL_LOGS_JSON("REPEATABLE_ALL_LOGS_JSON");
+        REPEATABLE_ALL_LOGS_JSON("REPEATABLE_ALL_LOGS_JSON"),
+
+        /**
+         * gke service account key file location
+         */
+        ACCESS_KEY_FILE_LOCATION("accessKeyFileLocation");
```

Comment:
ConfigurationContext is a generic file, hence is not about kubernetes. In that view, parameter key ACCESS_KEY_FILE_LOCATION has no meaning. Let's prefix this with more context.